Sources/Classes/core/View/MiniAppViewHandler.swift

```diff
@@ -867,3 +871,17 @@ extension MiniAppViewHandler: WKUIDelegate {
         UIApplication.topViewController()?.present(alertController, animated: true, completion: nil)
     }
 }
+
+// MARK: - Universal Bridge
+extension MiniAppViewHandler {
+    internal func sendJsonToMiniApp(string jsonString: String?) {
+        NotificationCenter.default.sendCustomEvent(
+            MiniAppEvent.Event(
+                miniAppId: appId,
+                miniAppVersion: version ?? "",
+                type: .miniappReceiveJsonString,
+                comment: jsonString ?? ""
+            )
```

Comment:
You can use self.didReceiveEvent(event.type, message: event.comment) right ?

Reply:
This will now be updated to `self.didReceiveEvent(MiniAppEvent.miniappReceiveJsonString, message: jsonString ?? "")` in the next commit.